### ISSUE TITLE
An argument named "auto_swap_slot_name" is not expected in azurerm_linux_function_app resource.

### ISSUE THREAD

**Issue by Remigijus-D** [hashicorp/terraform-provider-azurerm#16913]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.1.9

### AzureRM Provider Version

3.3.0

### Affected Resource(s)/Data Source(s)

azurerm_linux_function_app

### Terraform Configuration Files

```hcl
Error: Unsupported argument
│
│   on az-function-app\main.tf line 51, in resource "azurerm_linux_function_app" "main":
│   51:     auto_swap_slot_name                           = var.auto_swap_slot_name
│
│ An argument named "auto_swap_slot_name" is not expected here.
```

### Debug Output/Panic Output

```shell
N/A
```

### Expected Behaviour

Please remove argument "auto_swap_slot_name" from documentation of azurerm_linux_function_app resource.

**Comment by xiaxyi** [comment]
@Remigijus-D seems the property is not supported yet, were you able to use this property in the past?

**Comment by xiaxyi** [comment]
@Remigijus-D The doc is modified, may i know if you have any other question? feel free to let me know your concerns.

**Comment by rcskosir** [comment]
@Remigijus-D Thank you for opening this issue. Thank you @xiaxyi for modifying the documentation. Since this issue has been resolved with #17316, I am going to mark this issue as closed.
